wbooks/src/app/components/ButtonsApp/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Text, TouchableOpacity } from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
+import { colorsGradient } from '@constants/colorsApp';
+
+import styles from './styles';
+
+export function BtnPrimary({ title, onPress }: any) {
+  return (
+    <TouchableOpacity onPress={onPress}>
+      <LinearGradient
+        colors={colorsGradient}
+        style={styles.button}
+        start={{ x: 0, y: 0.5 }}
+        end={{ x: 1, y: 0.5 }}>
+        <Text style={styles.titleButton}>{title}</Text>
+      </LinearGradient>
+    </TouchableOpacity>
+  );
+}
+export function BtnSecondary({ title, onPress }: any) {
```

Comment:
Don't use `any` for the props here. Use cmd+click in VSCode to see the types of the props of `TouchableOpacity` and `Text`. Use those types to add custom types to your components.